src/components/MiniPlayer.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+import CloseIcon from "@/assets/svgs/CloseIcon.svg?react";
+import PauseIcon from "@/assets/svgs/PauseIcon.svg?react";
+import PlayIcon from "@/assets/svgs/PlayIcon.svg?react";
+
+const MiniPlayer = ({ thumbnail, channelName, closePlayer }) => {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const handlePlayPause = () => {
+    setIsPlaying((prev) => !prev);
+  };
+
+  return (
+    <div className="flex h-20 w-[390px] justify-between rounded-b-4xl bg-white px-4 shadow-[0_-6px_9px_rgba(0,0,0,0.3)]">
```

Comment:
```suggestion
    <div className="flex h-20 w-full justify-between rounded-b-4xl bg-white px-4 shadow-[0_-6px_9px_rgba(0,0,0,0.3)]">
```
현재 작성하신 코드대로라면 해당 컴포넌트를 가져와 사용할 때, width가 390px 이상인 화면에서는 양옆이 비어보일 것 같아서 width를 100%로 수정하면 좋을 것 같아요!

Reply:
말씀해주신 부분 반영해서 w-full로 수정했습니다!
감사합니다 😊